src/Casts/DateTimeInterfaceCast.php

```diff
@@ -24,8 +24,12 @@ public function cast(DataProperty $property, mixed $value): DateTimeInterface |
             return Uncastable::create();
         }
 
-        /** @var \DateTime|\DateTimeImmutable $name */
-        $datetime = $type::createFromFormat($format, $value);
+        /** @var \DateTime|\DateTimeImmutable $type */
+        try {
+            $datetime = $type::createFromFormat($format, $value);
+        } catch (\Throwable $e) {
```

Comment:
Import this one please 🙂

Reply:
Three of them? or just Throwable?
I've removed InvalidFormatException of Carbon library. Dunno how to make it cleaner.
I think thats ok, because we dont do much logic here, that can raise Exception outside of Datetime scope.

Reply:
`Throwable` must be

Reply:
Done